tools/response_file.py

```diff
@@ -13,13 +13,16 @@
 DEBUG = int(os.environ.get('EMCC_DEBUG', '0'))
 
 
-def create_response_file(args, directory):
+def create_response_file(args, directory, suffix='.rsp.utf-8'):
```

Comment:
Can we just leave this default as `.rsp` since the default is utf-8 anyway?

Alternatively why not take an "encoding" here which defaults to `utf-8`.

Then do:
```
suffix = '.rsp'
# For non-default (non-utf-8) encodings we append the encoding name to the file suffix
if encoding != 'utf-8':
  suffix += '.' + encoding
```

Then you could remove all the 6 lines below around `Decide the encoding ...` ?

Reply:
> Can we just leave this default as `.rsp` since the default is utf-8 anyway?

If we do this suffix-based convention, it would be preferable to switch the Emscripten-created response files to always advertise the encoding? No harm in doing that?

> Alternatively why not take an "encoding" here which defaults to `utf-8`.

If I do that, then if user calls

```py
create_response_file(args, directory, suffix=".foo", encoding="cp1252")
```
then user would not get a response file with suffix `.foo` that the function call seems to be asking, but a response file with suffix `.foo.cp1252`. That would not feel good for the API usage. I think it is better to have the user specify the suffix directly.

Reply:
My suggestion would be *not* to add the new `suffix` argument, just the optional encoding.  The suffix is not specify-able today so hopefully we don't need that argument.